tests/unit_tests/test_color_math.py

```diff
@@ -0,0 +1,22 @@
+from SyntheticDataset import ColorMath
+import unittest
+from math import sqrt
+
+class TestColorMath(unittest.TestCase):
```

Comment:
The name of the classes should be ThingToTestForTestCase(...). So, for instance, here you need ```ColorMathTestCase```